Content.Server/DeltaV/Station/Components/CaptainStateComponent.cs

```diff
@@ -0,0 +1,88 @@
+using Content.Server.DeltaV.Station.Systems;
+using Content.Server.Station.Systems;
+using Content.Shared.Access;
+using Robust.Shared.Prototypes;
+
+namespace Content.Server.DeltaV.Station.Components;
+
+/// <summary>
+/// Denotes a station has no captain and holds data for automatic ACO systems
+/// </summary>
+[RegisterComponent, Access(typeof(CaptainStateSystem), typeof(StationSystem))]
+public sealed partial class CaptainStateComponent : Component
+{
+    /// <summary>
+    /// Denotes wether the entity has a captain or not
+    /// </summary>
+    /// <remarks>
+    /// Assume no captain unless specified
+    /// </remarks>
+    [DataField]
+    public bool HasCaptain = false;
+
+    /// <summary>
+    /// Holds the round time of the last time a captain was present if one is not present currently
+    /// </summary>
+    [DataField]
+    public TimeSpan TimeSinceCaptain = TimeSpan.Zero;
+
+    /// <summary>
+    /// How long with no captain before an ACO is requested
+    /// </summary>
+    [DataField]
+    public TimeSpan ACORequestDelay = TimeSpan.FromMinutes(15);
+
+    /// <summary>
+    /// The message ID used for announcing the cancellation of ACO requests
+    /// </summary>
+    [DataField]
+    public string RevokeACOMessage = "captain-arrived-revoke-aco-announcement";
```

Comment:
locid

Reply:
locid as in changing the wording of the summary?

Reply:
or variable name?

Reply:
sorry I'm not quite sure what you mean

Reply:
Use LocId variables rather than strings for declaring localization